README.md

```diff
@@ -44,8 +44,7 @@ The Private Set Membership library requires the following dependencies:
 
 In order to run this library, you need to install Bazel, if you don't have
 it already.
-[Follow the instructions for your platform on the Bazel website. Make sure you
- are installing version 4.2.X.]
+[Follow the instructions for your platform on the Bazel website. This repo has been successfully built locally using version 6.1.2.]
```

Comment:
Maybe follow the same instructions as before:

"Make sure you are installing version 6.1.2."